lib/lita/adapters/slack/rtm_connection.rb

```diff
@@ -64,7 +64,7 @@ def send_messages(channel, strings)
         end
 
         def shut_down
```

Comment:
Yes, it is: 
https://github.com/litaio/lita/blob/e9abbce2aed0a6659e96250bb6a0b576ec306fdd/lib/lita/robot.rb#L129